torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -740,30 +780,15 @@ NvrtcFunction nvrtcCompile(
     // clamp down to register allocation granularity at warp level
     int effective_max_reg_per_warp = max_reg_per_warp /
         reg_allocation_granularity * reg_allocation_granularity;
-    int max_register =
-        std::min(effective_max_reg_per_warp / warp_size, max_regs_per_thread);
-
-    max_register_usage += std::to_string(max_register);
-    args.push_back(max_register_usage.c_str());
-  }
-
-  const char* ptxas_opt_level = getenv("PYTORCH_NVFUSER_JIT_OPT_LEVEL");
-  uint32_t jit_opt_level = 0;
+    max_register = static_cast<uint32_t>(
+        std::min(effective_max_reg_per_warp / warp_size, max_regs_per_thread));
```

Comment:
This isn't part of changes introduced in this PR but the math looks strange to me. Why would we divide `per_warp` by `warp_size`?

Reply:
effective_max_reg_per_warp / warp_size --> effective_max_reg_per_thread.

I guess it's the poor naming that makes this confusing.